ControlScheme.json

```diff
@@ -41,42 +41,42 @@
     },
     "left button on top of joystick (ThrustmasterT16000M)"
   ],
-  "elevatorAlgaeHigh": [
+  "algaeHigh": [
```

Comment:
is there a reason for having the same input 3 times? I see no difference in implementation between all 3